src/Ztm.WebApi/TransactionConfirmationWatcher.cs

```diff
@@ -0,0 +1,386 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using NBitcoin;
+using Ztm.Threading;
+using Ztm.Zcoin.Synchronization;
+using Ztm.Zcoin.Watching;
+
+namespace Ztm.WebApi
+{
+    using ConfirmContext = TransactionConfirmationWatch<TransactionConfirmationCallbackResult>;
+    using Timer = Ztm.Threading.Timer;
```

Comment:
We gonna use full path to type instead if we push it out of namespace scope. I think we should push it here to keep its short.

Reply:
It just the only place to have FQN. But what we get from it is consistency, which have benefits than shorter line. If we really want to put it here we should do it for the others too. The main problems with putting inside `namespace` is it will prefer the imported one over the one in the same namespace (see here for information https://stackoverflow.com/a/151560/1829232).